assets/pages/contributions-landing/components/contributionsContextIntro.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Component ----- //
+
+export default function ContributionsContextIntro() {
```

Comment:
Could this not go in `contributionsContext.jsx`?

Reply:
It's complicated, but it boils down to: CSS. These components have to flow in different ways at different breakpoints to fit the designs; they both have to be at the same level as the bundle, and they can't be wrapped together in the same div.

Currently React/Preact don't allow you to return arrays of elements from a component, they have to be wrapped in an enclosing element. I believe this is being introduced in [React Fiber](https://github.com/facebook/react/issues/2127), and it looks like [some work](https://github.com/developit/preact/pull/703) is happening for Preact as well.

Reply:
Fair enough